manifests/tftp/efi.pp

```diff
@@ -0,0 +1,66 @@
+# EFI support for TFTP
+class foreman_proxy::tftp::efi (
```

Comment:
The class name is little bit confusing, our initial goal was to support EFI but we also deploy ELF binaries for BOOTP (PPC) and BIOS firmware here. Better candidate would be perhaps "netboot".

Reply:
Ok, that makes sense.